python/pyproject.toml

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools",
+    "wheel",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../dependencies.yaml and run `rapids-dependency-file-generator`.
+
+[project]
+name = "cuxfilter"
+version = "23.08.00"
+description = "GPU accelerated cross filtering with cuDF"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    {name = "NVIDIA Corporation"},
+]
+license = { text = "Apache 2.0" }
+requires-python = ">=3.9"
+dependencies = [
+    "bokeh>=2.4.2,<=2.5",
+    "cudf==23.8.*",
+    "cupy-cuda11x>=12.0.0",
+    "cuspatial==23.8.*",
+    "dask-cuda==23.8.*",
+    "dask-cudf==23.8.*",
+    "datashader>=0.15",
+    "geopandas>=0.11.0",
+    "holoviews>=1.15.0,<=1.15.4",
+    "jupyter-server-proxy",
+    "libwebp",
+    "nodejs>=14",
```

Comment:
This seems incorrect. The PyPI package for `nodejs` isn't what you want here. https://pypi.org/project/nodejs/

I think `nodejs` is supposed to be a conda-only dependency.

Reply:
Probably the same issue exists for `libwebp` (no PyPI package, probably conda-only).

Reply:
that's a good point. I have moved it to conda specific requirements